cookbooks/lib/features/couchdb_spec.rb

```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
 
+def couchdb_url
+  return 'http://127.0.0.1:5984' if %w[trusty xenial].include?(Support.distro)
+  return 'http://admin:travis@127.0.0.1:5984' if 'bionic'.include?(Support.distro)
+
+  'http://127.0.0.1:5984'
```

Comment:
```suggestion
  'http://admin:travis@127.0.0.1:5984'
```

Reply:
I propose this as default return, we also have focal and probably more distros later